src/Appwrite/Statsd/Statsd.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Appwrite\Statsd;
+
+use Utopia\App;
+
+class Statsd
+{
+    /**
+     * @var array
+     */
+    protected $params = [];
+
+    /**
+     * @var mixed
+     */
+    protected $statsd;
+
+    /**
+     * Event constructor.
+     *
+     * @param mixed $statsd
+     */
+    public function __construct($statsd)
+    {
+        $this->statsd = $statsd;
+    }
+
+    /**
+     * @param string $key
+     * @param mixed  $value
+     *
+     * @return $this
+     */
+    public function setParam(string $key, $value): self
+    {
+        $this->params[$key] = $value;
+
+        return $this;
+    }
+
+    /**
+     * Save to statsd.
+     */
+    public function save(): void
```

Comment:
`save` is a bit misleading. The data is being submitted by UDP not really stored anywhere. `submit` would probably be a better name.